### ISSUE TITLE
Release of ShigaPass v1.5

### ISSUE THREAD

**Issue by ryanjameskennedy** [imanyass/ShigaPass#3]
Hi there,

I'm wondering of you could do a release of the latest version as a tar.gz? I want to add this to [bioconda-recipes](https://github.com/bioconda/bioconda-recipes) so that a singularity image is built and store in [galaxy project depot](https://depot.galaxyproject.org/singularity/). Appreciate the help.

**Comment by tseemann** [comment]
@ryanjameskennedy it looks like there is a tar.gz release for 1.5.0 now?

If you could solve Issue #5 that would be very helpfui!

**Comment by ryanjameskennedy** [comment]
Yeah this issue can be closed. I was offering to help in my spare time. I don't work for the same lab or the same country as Dr Yassine. Our hospital in Lund, Sweden, uses ShigaPass in our clinical microbiology WGS pipeline which uses singularity. So my first priority was to add it to galaxy project depot via bioconda-recipes. I would say that my PR is almost finished, though it definitely needs some validation tests. @imanyass are you able to compare my changes with a dataset?

**Comment by imanyass** [comment]
Sorry for my late reply, and thanks @ryanjameskennedy a lot for doing this! I can see that there’s now a Bioconda package for ShigaPass (https://anaconda.org/bioconda/shigapass), and I’m happy to test it on my side.
